CRM/Civicase/Hook/NavigationMenu/AlterForCaseMenu.php

```diff
@@ -39,8 +40,8 @@ public function run(array &$menu) {
   private function rewriteCaseUrls(array &$menu) {
     // Array(string $oldUrl => string $newUrl).
     $rewriteMap = [
-      'civicrm/case?reset=1' => 'civicrm/case/a/#/case?case_type_category=cases',
-      'civicrm/case/search?reset=1' => 'civicrm/case/a/#/case/list?sx=1',
+      'civicrm/case?reset=1' => CaseUrlHelper::getUrlByRouteType('dashboard'),
+      'civicrm/case/search?reset=1' => 'civicrm/case/a/#/case/list?sx=1&p=fn',
```

Comment:
Why did we add `fn`?

Reply:
This is an extra requirement added to the main task of changing the category name for ID. Is a parameter to be used later for breadcrumbs. 
I forgot initially to put it on the PR description, but now I have updated it.